cli/package.json

```diff
@@ -38,25 +38,25 @@
     "prettier": "^1.19.1",
     "rimraf": "^3.0.2",
     "semver": "7.3.2",
-    "table": "^5.4.6",
-    "unzipper": "^0.10.8",
+    "table": "^6.7.2",
+    "unzipper": "^0.10.11",
     "which": "^2.0.2",
-    "yargs": "^15.1.0"
+    "yargs": "^17.2.1"
```

Comment:
This is a large bump are we aware of the impact here or should it also be a separate PR?
Just trawling through changelogs I'm seeing a wall of text that I don't necessarily understand
https://github.com/yargs/yargs/blob/main/CHANGELOG.md#1600-2020-09-09